packages/http/libraries/core/src/http/typeguard/isApplyMiddlewareOptions.ts

```diff
@@ -7,6 +7,8 @@ export function isApplyMiddlewareOptions(
     value as ApplyMiddlewareOptions;
```

Comment:
Let's use the Partial here as well (see below comments at `isPipe`)

Reply:
Fixed